### PR TITLE
[ACS-3553] Fix process page accessibility from csv(880320,880281)

### PR DESCRIPTION
**Pull request by rbahirsheth**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**
Start process page Issues resolved from csv file issue ID 880320,880281

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services/src/lib/content-widget/attach-file-widget.component.html [hunk 1]
```diff
@@ -59,18 +59,17 @@
...
-        <mat-list-item class="adf-attach-files-row" *ngFor="let file of field.value">
+    <mat-list *ngIf="hasFile" role="list">
+        <mat-list-item class="adf-attach-files-row" *ngFor="let file of field.value" role="listitem">
             <img mat-list-icon class="adf-attach-widget__icon"
                  [id]="'file-'+file.id+'-icon'"
                  [src]="file.content ? getIcon(file.content.mimeType) : getIcon(file.mimeType)"
                  [alt]="mimeTypeIcon"
                  (click)="onAttachFileClicked(file)"
                  (keyup.enter)="onAttachFileClicked(file)"
-                 role="button"
```
Comment (MichalKinas) [inline lib/process-services/src/lib/content-widget/attach-file-widget.component.html:70]: Are we sure that removing this role is correct? These elements have some click and keyboard functions attached

Comment (rbahirsheth) [reply lib/process-services/src/lib/content-widget/attach-file-widget.component.html:70]: Yes i saw click event are there for image & the file name too. But still click event is working.

Comment (DenysVuika) [reply lib/process-services/src/lib/content-widget/attach-file-widget.component.html:70]: the role is not for clicks, it's for screen readers to find where the button is, whether it is there or not, the click events are not related to that

Comment (rbahirsheth) [reply lib/process-services/src/lib/content-widget/attach-file-widget.component.html:70]: Addressed comment. please check.

Comment (DenysVuika) [reply lib/process-services/src/lib/content-widget/attach-file-widget.component.html:70]: why it's needed to touch the "role" at all? what does it fix?

Comment (rbahirsheth) [reply lib/process-services/src/lib/content-widget/attach-file-widget.component.html:70]: Machine is reading as 'undefined' button. We have image tag so i have used role img & aria label.

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**